DOCSUP-10296: Document the toJSONString function

lehasm opened the pull request:
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #25164

lehasm committed:
Draft

lehasm committed:
Ready for PR draft

lehasm committed:
Minor update

adevyatova commented on docs/en/sql-reference/functions/json-functions.md:
```diff
@@ -306,3 +306,43 @@ Result:
+
+**Example**
+
+The first example serializes a [map](../../sql-reference/data-types/map.md).
```
```suggestion
The first example serializes a [Map](../../sql-reference/data-types/map.md).
```

lehasm committed:
Update docs/en/sql-reference/functions/json-functions.md

Co-authored-by: Anna <42538400+adevyatova@users.noreply.github.com>

lehasm committed:
Update docs/en/sql-reference/functions/json-functions.md

Co-authored-by: Anna <42538400+adevyatova@users.noreply.github.com>

lehasm committed:
More fixes

lehasm committed:
Removed warning that Map is expetimental
and one must set allow_experimental_map_type = 1 to use it

lehasm committed:
The only example left
'quoted' changed to 'enclosed in qoutes'
Added See Also with link to related settings.

lehasm committed:
output_format_json_quote_64bit_integers updated

lehasm committed:
Перевод

lehasm committed:
Second example returned.
output_format_json_quote_denormals mentioned in the description.

lehasm commented:
@CurtizJ, please have a look.
